index.html

```diff
@@ -8,9 +8,87 @@
+    <link rel="stylesheet" href="css/style.css">
   <title>Responsive</title>
 </head>
 <body>
-  <!-- Your code here! Good luck! -->
+  <div id="wrapper">
+    <header>
+      <div class="container">
+        <div class="header-wrap">
+          <div class="logo">
+            <a href="#">
+              <img src="img/logo.png" alt="logo">
+            </a>
+          </div>
+          <nav class="nav">
+            <ul>
+              <li>
+                <a href="#">HOME</a>
+              </li>
+              <li>
+                <a href="#section about">ABOUT</a>
+              </li>
+              <li>
+                <a href="#">SERVICES</a>
+              </li>
+              <li>
+                <a href="#">PRICING</a>
+              </li>
+              <li>
+                <a href="#">CONTACT</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="container">
+          <div class="section-hero-wrapper" id="hero-section">
+              <section class="hero">
+                <h1>Proffesional Developers Health Care</h1>
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptates qui, neque minima quibusdam, necessitatibus id fuga</p>
+                <div class="button-holder">
+                  <button class="button">
+                    <i class="fas fa-heartbeat"></i>
+                  </button>
+                  <button class="button">
+                    <i class="fas fa-first-aid"></i>
+                  </button>
+                  <button class="button">
+                    <i class="fas fa-stethoscope"></i>
+                  </button>
+                </div>
+              </section>
+          </div>
+          <div class="about-section-wrapper">
+            <div class="image-holder">
+              <img src="img/tools.png" alt="tools">
+            </div>
+            <div class="about-holder" id="section about">
+              <section class="about">
+                <h2>About Clinic</h2>
+                <a href="#hero-section">PROFESSIONAL CARE</a>
+                 <h4><a href="#"><i class="fas fa-long-arrow-alt-right"></i> Medical Consulting</a></h4>
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Excepturi maxime esse velit tempore nobis</p>
+                <h4><a href="#"><i class="fas fa-long-arrow-alt-right"></i> Laboratory analysis</a></h4>
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Excepturi maxime esse velit tempore nobis</p>
+                <h4><a href="#"><i class="fas fa-long-arrow-alt-right"></i> Diagnostics</a></h4>
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Excepturi maxime esse velit tempore nobis</p>
+              </section>
+            </div>
+          </div>
+            <form action="#hero-section" method="get">
```

Comment:
Do you really need a method and specific action here? If not you can use `#`.

Reply:
don`t need